mlrun/api/api/endpoints/frontend_spec.py

```diff
@@ -82,8 +82,9 @@ def get_frontend_spec(
         default_function_pod_resources=mlrun.mlconf.default_function_pod_resources.to_dict(),
         default_function_preemption_mode=mlrun.mlconf.function_defaults.preemption_mode,
         feature_store_data_prefixes=config.feature_store.data_prefixes.to_dict(),
+        # ce_mode is deprecated, we will use the full ce config instead
```

Comment:
add a comment about the version that it will be deprecated ( if we are working on 1.3.0 then will be removed in 1.6.0)